### PR TITLE
Added some guidelines

### PR DESCRIPTION
**Pull request by KeizerDev**
The `1.` is very personal but I do like to see that so we won't get a mess of commits for multiple providers.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# S Contributing Guidelines
+
+## Rules
+
+There are a few basic ground-rules for contributors to `s`:
+
+1. When adding a `provider` use for each a **new single** commit.
```
Comment (zquestz) [inline CONTRIBUTING.md]: NIT: new **single**

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+2. Use gofmt on all files.
```
Comment (zquestz) [inline CONTRIBUTING.md]: Also would say that for refactor commits to base code that it should be discussed prior to PR.

**Comment by KeizerDev** [comment]
@zquestz Updated.

**Comment by zquestz** [comment]
LGTM

**Comment by zquestz** [comment]
BTW appreciate all the help on the project!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# S Contributing Guidelines
+
+## Rules
+
+There are a few basic ground-rules for contributors to `s`:
+
+1. When adding a `provider` use for each a new **single** commit.
+2. Use gofmt on all files.
+3. Refactor commits to the base code should be discussed prior to PR.
```
Comment (mbhinder) [inline CONTRIBUTING.md]: Add "in an issue" ?

Comment (zquestz) [inline CONTRIBUTING.md]: Like it.

**Comment by KeizerDev** [comment]
Done.

**Comment by zquestz** [comment]
Fee free to merge, you got an LGTM, don't want to steal your first merge opportunity. =)

**Comment by mbhinder** [comment]
Thanks LGTM

**Comment by KeizerDev** [comment]
Lol, here I go :smile:

**Pull request merged**